docs/theme/head.hbs

```diff
@@ -0,0 +1,13 @@
+    <meta httpEquiv="Content-Language" content="en-US">
+    <meta name="apple-mobile-web-app-title" content="Starknet Foundry Book">
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@swmansionxyz">
+    <meta name="twitter:creator" content="@jajakobyly">
```

Comment:
Can we skip this tag?

Reply:
I have to research, because it's only copied from scarb